metricbeat/module/dropwizard/collector/collector_integration_test.go

```diff
@@ -67,19 +71,14 @@ func TestFetch(t *testing.T) {
 	}
 	assert.Equal(t, hasTag, true)
```

Comment:
Do we only need to check hasTag once after all for _, event := range events {} loops?

Reply:
Uhmmm :thinking: You are right. I'll leave it for a follow up PR, I'm looking forward to finish with the migration and this was like this already anyways :sweat_smile: